client/traceidinjector.go

```diff
@@ -33,5 +33,5 @@ func generateFallBackTraceId() string {
 }
 
 func WithTracingInterceptor(logger log.Logger) grpc.DialOption {
```

Comment:
why do we need to have this here ? can we just expose the interceptor and let the client/user handle what he wants to do with this?
This limits chaining interceptors at the client side. @trivishnu

Reply:
+1

Reply:
removed as per discussion with @trivishnu .